exercises/square-root/description.md

```diff
@@ -0,0 +1,9 @@
+Given a natural radicand, return its square root.
+
+Check out the Wikipedia pages on [square root](https://en.wikipedia.org/wiki/Square_root) and [methods of computing square roots](https://en.wikipedia.org/wiki/Methods_of_computing_square_roots).
+
+Note that the term "radicand" refers to the number for which the root is to be determined. That is, it is the number under the root symbol.
+
+Recall also that natural numbers are positive real whole numbers (i.e. 1, 2, 3 and up).
+
+When implementing this in a language with a built-in square root function, implement your own function.
```

Comment:
it looks like we are taking a different strategy vs https://github.com/exercism/problem-specifications/pull/1418 here. In that PR, we explicitly chose to remove a sentence that says "if your language already has this, instead implement it yourself". In this PR, we choose to leave such a sentence in. There is probably a unifying principle that ties the two decisions together. It may be good to state it.

This comment should not be taken as a recommendation for or against any particular decision.

Reply:
Hmmm. I understand the point being made on that PR. But disagree with it.
`sqrt()` is something that is present in libraries in a lot of languages. But so is a `list` object yet we have `list-ops`, so is base conversion yet we have `all-your-base`, so are BSTs, yet we have `binary-search-tree`, etc etc.

If we reason that exercises should allow using the language features that they are intended to teach then a very large amount of the exercises become one liners across a lot of tracks.
On the other hand removing such exercises results in a much reduced set. 

The one option left? Suggest that learners do not use those language features in such exercises. Seems reasonable if someone truly wants to learn. And there is nothing preventing them from using the features if they don't. In this way the line is more a confirmation of the intent of the exercise rather than some kind of rule.

Reply:
Am I correct in assuming that the discussion is still open on what we want to do here?

Reply:
Going with the idea in #1418, and the fact that that PR was merged, I have removed the line from the description here so as to follow suit.